src/main/java/com/shopify/model/fulfillmentOrderApi/ShopifyDeliveryMethod.java

```diff
@@ -0,0 +1,101 @@
+package com.shopify.model.fulfillmentOrderApi;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+import org.joda.time.DateTime;
+
+import com.shopify.model.adapters.DateTimeAdapter;
+
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.FIELD)
+public class ShopifyDeliveryMethod {
+
+	public enum MethodType {
+		// No delivery method.
```

Comment:
Can you make these javadocs instead of plain comments. I think it would be beneficial to end users to understand.